maroon/src/network/p2p.rs

```diff
@@ -213,12 +205,12 @@ fn handle_receiver_outbox(
 }
 
 fn handle_swarm_event(
-  counter_requests: &Counter<u64>, swarm: &mut Swarm<MaroonBehaviour>, event: SwarmEvent<MaroonEvent>,
-  to_app: &UnboundedSender<Inbox>, alive_peer_ids: &mut HashSet<PeerId>, alive_gateway_ids: &mut HashSet<PeerId>,
+  swarm: &mut Swarm<MaroonBehaviour>, event: SwarmEvent<MaroonEvent>, to_app: &UnboundedSender<Inbox>,
```

Comment:
Wait, shouldn't these be one argument per line? I'm sure my `rustfmt` would make it so.

Reply:
No, we have [fn_params_layout = "Compressed"](https://github.com/rust-lang/rustfmt/blob/master/Configurations.md#compressed-1) but we can change it to Tall or Vertical. If you want strictly one parameter per line - it's Vertical